public/javascripts/Admin/src/Admin.js

```diff
@@ -69,6 +69,21 @@ function Admin(_, $, difficultRegionIds) {
         self.adminGSVLabelView = AdminGSVLabelView(true);
     }
 
+    function initializeAdminGSVCommentView(){
+        self.adminGSVCommentView = AdminGSVCommentView(true);
+    }
+
+    function initializeAdminGSVCommentWindow(){
+        $('.show-comment-location').click(function(e) { 
+            e.preventDefault();
+            var heading = parseFloat($(this).data('heading'));
```

Comment:
I was running into a bug frequently where the GSV imagery wouldn't show up even when there was imagery for it. I was seeing an error that said something like "`a.heading.toFixed` is not a function". I eventually figured out that it was because when you're reading the data using `$(this).data('heading')`, it usually comes back as a string. So I just parsed all these values to the appropriate data type, and it's all working now :ok_hand: